airflow-core/src/airflow/ui/src/i18n/locales/pl/components.json

```
@@ -93,6 +105,12 @@
     "selectDescription": "Wyzwól pojedyncze wykonanie tego Daga",
     "selectLabel": "Pojedyncze wykonanie",
     "title": "Uruchom Daga",
+    "toaster": {
+      "success": {
+        "description": "Dag został pomyślnie uruchomiony.",
+        "title": "Uruchomienie Daga"
```

```suggestion
        "description": "Dag został pomyślnie uruchomiony.",
        "title": "Uruchomienie Daga"
```
This one is also tricky. I think to be consistent we should translate it as `Dag został pomyślnie wykonany.` but it suggest it already finished, so I think "Uruchomienie" here may be good.

---

Yeah. I am using "uruchomienia" to show current/future run, but when it's done, it's "wykonany". It's tricky but I started to feel a good approach here - > future/current = "uruchomienie", past = 'wykonanie".

---

Yeah. I am using "uruchomienia" to show current/future run, but when it's done, it's "wykonany". It's tricky but I started to feel a good approach here - > future/current = "uruchomienie", past = 'wykonanie".